server/posts/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from posts import views
+
+app_name = 'posts'
+
+urlpatterns = [
+    path('<uuid:author_id>/posts/', views.CreatePostView.as_view(), name='create'),
+    path('<uuid:author_id>/posts/<uuid:pk>', views.UpdatePostView.as_view(), name='update')
```

Comment:
does the end point crash of author_id provided is not in the format of UUID? I use slug just in case user does not provide it as UUID format